### PR TITLE
Add ssl_endpoint_identification_algorithm configuration option

### LINKED ISSUE #298: Support `ssl.endpoint.identification.algorithm` property
TLS endpoint verification checks are enabled/disabled in Kafka via the `ssl.endpoint.identification.algorithm` property. 

The default for this value was changed from '' to `https` for Kafka 2.0, turning on TLS endpoint verification by default, potentially breaking existing installations that did not require endpoint verification.

### PR DESCRIPTION
**Pull request by robbavey**
Fixes #298

### CODE REVIEW

**Comment by robbavey** [comment]
@karenzone Would you mind checking the doc changes here?

**Comment by robbavey** [comment]
@karenzone Would you mind checking the doc changes here?

File: docs/index.asciidoc [hunk 1]
```diff
@@ -462,6 +463,15 @@ The size of the TCP send buffer (SO_SNDBUF) to use when sending data
...
 and a rebalance operation is triggered for the group identified by `group_id`
 
+[id="plugins-{type}s-{plugin}-ssl_endpoint_identification_algorithm"]
+===== `ssl_endpoint_identification_algorithm`
+
+  * Value type is <<string,string>>
+  * Default value is `"https"`
+
+The endpoint identification algorithm, defaults to "https". Set to empty string "" to disable endpoint verification
+
```
Comment (karenzone) [inline docs/index.asciidoc]: Please put `"https"` and `""` in backticks.  
Otherwise, builds cleanly and  LGTM. Thanks!

Comment (robbavey) [reply docs/index.asciidoc]: Done - thank you @karenzone!

**Pull request closed without merging**